$NetBSD: patch-ab,v 1.4 2012/08/20 13:23:18 imilh Exp $

Leaving this line in leads to a compile error in openssl's crypto.h

--- src/pcap.h.orig	2009-06-14 00:49:09.000000000 +0200
+++ src/pcap.h
@@ -24,7 +24,8 @@
 #define uchar  unsigned char
 #define ushort unsigned short
 #define uint   unsigned int
-#define ulong  unsigned long
+// leaving this line in leads to a compile error in openssl's crypto.h
+//#define ulong  unsigned long
 
 //BSSID const. length of 6 bytes; can be together with all the other types
 #define IVS2_BSSID	0x0001
